reconcile/cli.py

```diff
@@ -1968,12 +1968,18 @@ def template_validator(ctx):
 @integration.command(short_help="Render datafile templates in app-interface.")
 @click.option(
     "--app-interface-data-path",
-    help="Path to app-interface dictory, used to write output and calculate diff in dry-run.",
+    help="Path to data dir in app-interface repo. Use this for local rendering or in MR checks.",
     required=False,
     envvar="APP_INTERFACE_DATA_PATH",
 )
+@click.option(
+    "--git-clone-path",
```

Comment:
I thought this is the git url of the repo, but later I found that is read from settings, then why do we need a path here since we always use a new temp dir? Maybe change it to a flag to indicate whether use local or do a git clone.

Reply:
Yes, that was about the last reason I had for this parameter. You are right, I'll change it